webapp/javascript/redux/reducers/settings.ts

```diff
@@ -15,25 +16,36 @@ import {
   createAPIKey as createAPIKeyAPI,
   deleteAPIKey as deleteAPIKeyAPI,
 } from '@webapp/services/apiKeys';
+import { fetchApps, deleteApp as deleteAppAPI } from '@webapp/services/apps';
 import type { RootState } from '@webapp/redux/store';
 import { addNotification } from './notifications';
 import { createAsyncThunk } from '../async-thunk';
 
-type UsersState = {
-  type: 'pristine' | 'loading' | 'loaded' | 'failed';
-  data?: Users;
-};
+enum FetchStatus {
```

Comment:
enum keys and values are the same, maybe it's better to use Numeric enums, instead of String ones?

Reply:
Good point, but we are using the same strings statuses in other reducers and I think we should refactor them all in the other task